src/WebJobs.Script/Description/DotNet/DependencyHelper.cs

```diff
@@ -0,0 +1,108 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.Extensions.DependencyModel;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Script.Description
+{
+    public static class DependencyHelper
```

Comment:
Please add detailed description text for this class and important methods

Reply:
Though I see your one method has help text below

Reply:
The relevant (and public) one was documented. The other are supporting methods here.